install.sh

```diff
@@ -206,6 +212,10 @@ fi
 cd "/usr" || exit 1
 
 ####################################################################### script
+if should_install_git; then
+    abort "You must install Git before installing Homebrew."
```

Comment:
Should this link to https://docs.brew.sh/Installation?

Reply:
Great idea. Added link.